Marks activateService and activateServicePlan public APIs as deprecated. (#15534)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/organization-activateservice.md

doc_type: apiPageType
---

# organization: activateService (deprecated)

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> [!CAUTION]

> > The organization `activateService` API will be deprecated and will stop returning data on March 15, 2022.

Reviewer: ```suggestion
> The **activateService** API is deprecated and will stop returning data on March 15, 2022.
```

|Content-Type|application/json. Required.|

## Request body

> In the request body, supply a JSON representation of the [activateService](../resources/activateService.md) (deprecated) object.

Reviewer: No need to indicate this here. The titles (API + resource) are sufficient.
```suggestion
In the request body, supply a JSON representation of the [activateService](../resources/activateService.md) object.
```
